xbmc/platform/linux/WebOSTVPlatformConfig.h

```diff
@@ -8,13 +8,33 @@
 
 #pragma once
 
+#include <atomic>
+#include <memory>
+
+#include <webos-helpers/libhelpers.h>
+
 class CVariant;
 
+struct HContextDeleter
+{
+  void operator()(HContext* ctx) const noexcept
+  {
+    if (ctx)
+      HUnregisterServiceCallback(ctx);
+  }
+};
+
 class WebOSTVPlatformConfig
 {
+private:
+  static inline std::unique_ptr<HContext, HContextDeleter> m_requestContextARC{new HContext{}};
+  static inline std::atomic<bool> m_eAC3Supported;
```

Comment:
Initialize with `false`

```suggestion
  static inline std::atomic<bool> m_eAC3Supported{false};
```